packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/bin/server/services/users_service.dart

```diff
@@ -67,4 +72,30 @@ class UsersService {
     createUser(newUser);
     return newUser;
   }
+
+  void setPasswordForUser(String email, String password) =>
+      _usersRepository.setPasswordForUser(email, password);
+
+  String? getPasswordForUser(String email) =>
+      _usersRepository.getPasswordForUser(email);{{#enable_forgotten_password}}
+
+  bool isPasswordResetLockedForUser(String email) =>
+      _usersRepository.isPasswordResetLockedForUser(email);
+
+  int getPasswordResetTimeoutForUser(String email) =>
+      _usersRepository.getPasswordResetTimeoutForUser(email);
+
+  void lockPasswordResetForUser(String email) {
+    _usersRepository.lockPasswordResetForUser(email);
+    Timer.periodic(
+      const Duration(seconds: 1),
+      (timer) {
+        if (timer.tick == 60) {
```

Comment:
The `60` seconds value seems arbitrary. Can we introduce a constant so that we have a unique place where we can configure this, instead going manually over each occurrence?